pkgs/by-name/ht/http3-ytproxy/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, libwebp
+}:
+buildGoModule rec {
+  pname = "http3-ytproxy";
+  version = "unstable-2022-07-03";
+
+  src = fetchFromGitHub {
+    owner = "TeamPiped";
+    repo = pname;
+    rev = "4059da180bb9d7b0de10c1a041bd0e134f1b6408";
+    hash = "sha256-ilIOkZ9lcuSigh/mMU7IGpWlFMFb2/Y11ri3659S8+I=";
+  };
+
+  postPatch = ''
+    substituteInPlace main.go \
+      --replace 'lucas-clemente/quic-go' 'quic-go/quic-go'
```

Comment:
We should do this via a patch. Go is rather particular about it's files.

Reply:
I've changed this dependency update, the go bump and running `go mod tidy` to be applied through a patch.